microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -247,15 +249,15 @@ The updated `manifest.json` that includes the `content-scripts` and `web_accessi
 
 ---
 
-The section you added is `content_scripts`.  The `matches` attribute is set to `<all_urls>`, which means that all files in `content_scripts` are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript and CSS.  You also added `lib\jquery.min.js`.  You're able to include that from the download mentioned at the top of the section.
+The `matches` attribute is set to `<all_urls>`, which means that all files in `content_scripts` are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript and CSS.  You also added `lib\jquery.min.js`.  You're able to include that from the download mentioned at the top of the section.
 
-6. Add jQuery and understanding the associated thread
+## Add jQuery and understanding the associated thread
```

Comment:
```suggestion
<!-- ====================================================================== -->
## Add jQuery and understanding the associated thread
```

Reply:
Consider two separate headings, or some way to avoid the compound asymmetrical heading: maybe:
h2 Add jQuery to XYZ ... the associated thread
h4 Understanding the associated thread